test/integration/default/default_spec.rb

```diff
@@ -14,3 +14,7 @@
   it { should be_listening }
   its('protocols') { should include 'tcp' }
 end
+
+describe command('curl -fs http://localhost:8080/metrics/find?query=carbon.*') do
```

Comment:
Can we convert this to a http resource now that those run on the host

Reply:
@jtschelling Please address this